.changeset/polite-maps-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rainbow-me/rainbowkit": patch
+---
+
+Implemented logout on wallet switch in `<RainbowKitAuthenticationProvider />`
```

Comment:
Might say something more specific like "Fixed an issue where a user would not automatically be logged out after switching their wallet in MetaMask or other browser wallets"